pytorch-cluster.spec

```diff
@@ -16,6 +16,12 @@ Requires: py3-torch %{!?without_cuda:cuda}
 
 %prep
 %setup -n %{n}-%{realversion}
+#Set cms_cxx_standard to 20 as CUDA 12 doesn't yet support C++23
+%if "%{cms_cxx_standard}" == "23"
+%undefine cms_cxx_standard
+%define cms_cxx_standard 20
+%endif
+
```

Comment:
@iarspider , as we will build CPP23 IBs without `cuda` to this condition is not needed.  Can you remove this condition from all `pytorch-*` spec